### PR TITLE
Add sm_dealdamage_model_enable convar

### PR DESCRIPTION
**Pull request by 8guawong**
Add sm_dealdamage_model_enable convar to disable changing player model

### CODE REVIEW

File: addons/sourcemod/scripting/MyJailbreak/hide.sp [hunk 1]
```diff
@@ -1173,7 +1269,7 @@ void CreateInfoPanel(int client)
 	Format(info, sizeof(info), "%T", "hide_info_title", client);
 	InfoPanel.SetTitle(info);
 
-	InfoPanel.DrawText("                                   ");
+	InfoPanel.DrawText("								   ");
```
Comment (shanapu) [inline addons/sourcemod/scripting/MyJailbreak/hide.sp]: small mistake.
IIRC the tabs will be shown as square.

**Comment by shanapu** [comment]
Much thanks for your work on this PR! 
There is a small mistake by search and replace spaces to tabs. please correct this and I will merge you request.
And don't forget to add yourself as contributor to the headers like https://github.com/shanapu/MyJailbreak/blob/master/addons/sourcemod/scripting/MyJailbreak/ratio.sp#L7-L8

File: addons/sourcemod/scripting/MyJailbreak/dealdamage.sp [hunk 2]
```diff
@@ -205,6 +207,7 @@ public void OnPluginStart()
 	gc_bChat = AutoExecConfig_CreateConVar("sm_dealdamage_chat", "1", "0 - disabled, 1 - enable print results in chat", _, true, 0.0, true, 1.0);
 	gc_bConsole = AutoExecConfig_CreateConVar("sm_dealdamage_console", "1", "0 - disabled, 1 - enable print results in client console", _, true, 0.0, true, 1.0);
 	gc_bColor = AutoExecConfig_CreateConVar("sm_dealdamage_color", "1", "0 - disabled, 1 - color the model of the players", _, true, 0.0, true, 1.0);
+	gc_bModel = AutoExecConfig_CreateConVar("sm_dealdamage_model_enable", "1", "0 - disabled, 1 - enable model ", _, true, 0.1, true, 1.0);
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/dealdamage.sp]: Why 0.1 instead of 0.0?

File: addons/sourcemod/scripting/MyJailbreak/dealdamage.sp [hunk 3]
```diff
@@ -1189,7 +1201,10 @@ void PrepareDay(bool thisround)
 			MyIcons_BlockClientIcon(i, true);
 		}
 
-		CreateTimer (1.1, Timer_SetModel, i);
+		if (gc_bModel.BoolValue)
+		{
+			CreateTimer (1.1, Timer_SetModel, i);
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/dealdamage.sp]: We should pass the client userid. And maybe in the callback check if the player still alive or not?

File: addons/sourcemod/scripting/MyJailbreak/hide.sp [hunk 4]
```diff
@@ -760,6 +814,42 @@ public void Event_TA_Detonate(Event event, const char[] name, bool dontBroadcast
...
+	int decreaseHP = gc_iHPSeekerDec.IntValue;
+	int clientHealth = GetClientHealth(client);
+
+	if ((clientHealth - decreaseHP) > 0) 
+	{
+		SetEntityHealth(client, (clientHealth - decreaseHP));
+	} 
+	else 
+	{
+		CreateTimer(0.1, Timer_SlayClient, client, TIMER_FLAG_NO_MAPCHANGE);
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/hide.sp]: Pass the userid here as well

File: addons/sourcemod/scripting/MyJailbreak/dealdamage.sp (hunk 2 above)
Comment (shanapu) [reply addons/sourcemod/scripting/MyJailbreak/dealdamage.sp]: yes, should be 0.0

File: addons/sourcemod/scripting/MyJailbreak/dealdamage.sp (hunk 3 above)
Comment (shanapu) [reply addons/sourcemod/scripting/MyJailbreak/dealdamage.sp]: I missed that. Yes always pass userid on timer

**Comment by shanapu** [comment]
Just seen, this PR goes to master. 
Could you please send this PR to dev branch?

File: addons/sourcemod/scripting/MyJailbreak/hide.sp (hunk 4 above)
Comment (Hexer10) [reply addons/sourcemod/scripting/MyJailbreak/hide.sp]: Afaik is a good practice to  always do that

**Comment by Hexer10** [comment]
I think this is ready to be merged! waiting for @shanapu reply.

**Comment by shanapu** [comment]
Thank you all very much!

**Pull request merged**